MODULE.bazel

```diff
@@ -1,18 +1,129 @@
-module(name = "dre", version = "0.0.1")
+module(name = "dre")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "aspect_bazel_lib", version = "2.11.0")
+
+bazel_dep(name = "rules_python", version = "0.37.0")
+python_version = "3.12"
 
-bazel_dep(name="rules_python", version="0.37.0")
 python = use_extension("@rules_python//python/extensions:python.bzl", "python")
-python.toolchain(python_version = "3.12.2", is_default = True)
+python.toolchain(python_version = python_version)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+pip.parse(
+    hub_name = "python_deps",
+    python_version = python_version,
+    requirements_lock = "//:requirements.lock",
+)
+use_repo(pip, "python_deps")
 
 bazel_dep(name = "rules_mypy", version = "0.22.0")
 
-#types = use_extension("@rules_mypy//mypy:types.bzl", "types")
-#types.requirements(
-#    name = "pip_types",
-#    # `@pip` in the next line corresponds to the `hub_name` when using
-#    # rules_python's `pip.parse(...)`.
-#    # pip_requirements = "@pip//:requirements.bzl",
-#    # also legal to pass a `requirements.in` here
-#    requirements_txt = "//:requirements.lock",
-#)
-#use_repo(types, "pip_types")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_oci", version = "2.0.0")
+
+oci = use_extension("@rules_oci//oci:extensions.bzl", "oci")
+oci.pull(
+    name = "bazel_image_6_5_0",
+    image = "gcr.io/bazel-public/bazel@sha256:7506002de2e90431e194f8391f4600b57cf229280002b2fc473b88a95d7104bc",
```

Comment:
Should this also be bumped? I see there is a newer 7.x image at gcr.io/bazel-public/bazel

Reply:
That is a great catch. If we use bazel to do some queries in our `git` repo in release controller runtime this will break. I will bump it